### ISSUE TITLE
Add option to enable/disable tool tips in Administrator

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#2016]
It would be nice to have the ability to turn off tool tips on mouseovers.  This could be a checkbox on the Administrator tab of the Settings panel.

Imported Issue. Original Details:
Jira Issue Key: MIRTH-2055
Reporter: ericb
Created: 2012-01-13T11:33:03.000-0800

**Comment by rbeckman-nextgen** [comment]
Why are you looking to turn off tool tips?  Do you find that they are getting in the way?

Imported Comment. Original Details:
Author: jacobb
Created: 2013-10-17T17:07:01.000-0700

**Comment by rbeckman-nextgen** [comment]
Occasionally, yes.  I'll sometimes start typing in a field, and because of where I last left my mouse, what I'm typing won't be visible because it's blocked by a tool tip.  So I'll have to stop typing and move my mouse.

It's certainly not a big deal, nor should it be anywhere close to high priority, but it's just one of those nice little features to have (the ability to disable them) if you don't need them.  It would also nicely line up with the Appliance CP's ability to turn them off as well.

Imported Comment. Original Details:
Author: ericb
Created: 2013-10-18T14:28:48.000-0700
